src/test/MsQuicTests.h

```diff
@@ -780,6 +781,11 @@ typedef struct {
     QUIC_CTL_CODE(15, METHOD_BUFFERED, FILE_WRITE_DATA)
     // int - Family
 
+#define IOCTL_QUIC_RUN_TEST_ADDR_FUNCTIONS \
+    QUIC_CTL_CODE(16, METHOD_BUFFERED, FILE_WRITE_DATA)
```

Comment:
This must be added to the end of the file/list. There is already a `16` defined below.